main.go

```diff
@@ -18,5 +23,9 @@ func main() {
 		log.Fatalf("DISCORD_BOT_TOKEN must be set")
 	}
 	AutoLoadSave = !(os.Getenv("GROL_DISABLE_AUTOSAVE") == "1")
+	memlimit := debug.SetMemoryLimit(-1)
```

Comment:
Sane remarka comment would help as this method is uncommon

Reply:
also have to know (it's in the doc too) that the default value is MaxInt64